### PR TITLE
Add --superdigest cli argument

### PR DESCRIPTION
**Pull request by SgtPepperLHCB**

### CODE REVIEW

File: src/main/kotlin/zkr/ZkrOptions.kt [hunk 1]
```diff
@@ -21,6 +21,13 @@ class ZkrOptions {
     )
     var host: String = "localhost:2181"
 
+    @CommandLine.Option(
+            names = ["--superdigest"],
+            description = ["ZooKeeper superdigest password"],
+            defaultValue = ""
```
Comment (dylanmei) [inline src/main/kotlin/zkr/ZkrOptions.kt]: This option should be something that can be set in an environment variable

**Comment by dylanmei** [comment]
This should be represented on the README.

File: src/main/kotlin/zkr/ZkClient.kt [hunk 2]
```diff
@@ -38,6 +38,10 @@ class ZkClient(val host: String, val connect: Boolean = true, sessionTimeoutMill
                     throw IOException("Timeout out connecting to: $host")
                 }
                 logger.debug("connected")
+                if (superdigest.isNotBlank()) {
+                    logger.info("superdigest")
```
Comment (dylanmei) [inline src/main/kotlin/zkr/ZkClient.kt]: This can be removed.

File: src/main/kotlin/zkr/ZkrOptions.kt (hunk 1 above)
Comment (SgtPepperLHCB) [reply src/main/kotlin/zkr/ZkrOptions.kt]: Can't you just do `zkr --superdigest=$MY_ENV_VAR` when invoked? Or do you mean reading the System property `zookeeper.DigestAuthenticationProvider.superDigest` if it exists?  This is set something like `java -jar -Dzookeeper.DigestAuthenticationProvier.superDigest=super:blahblahblah`.

Comment (dylanmei) [reply src/main/kotlin/zkr/ZkrOptions.kt]: This is easy to do when running manually, but its more akward when running from a file such as a systemd unit.

Also, it exposes my password to `ps -aux` and other process inspection tools. I'm only recommending this (or `stdin`: see `docker login --help`) for passwords.

Comment (dylanmei) [reply src/main/kotlin/zkr/ZkrOptions.kt]: I see that you've removed the argument altogether.

Actually, I find that picocli has a great mechanism for this scenario: https://picocli.info/#_default_values_for_custom_variables

```
    @CommandLine.Option(
            names = ["--superdigest"],
            description = ["ZooKeeper superdigest password"],
            defaultValue = "${env:ZKR_SUPERDIGEST_PASSWORD}")
```

What do you think?

Also, I recommend using the ZKR_ prefix to deflect any surprising collisions in users' environments.

**Comment by dylanmei** [comment]
:+1: LGTM

**Pull request merged**